_config.yml

```diff
@@ -35,7 +35,7 @@ toc:
 # Tracker
 analytics: UA-89019737-3
 
-highlighter: none
+highlighter: rouge
```

Comment:
Hmm, I intentionally turned this off because not having highlights makes it easier to show diffs relative to previous code blocks. Let's throw some ideas around later